apps/judicial-system/web/src/components/AccordionItems/CaseFilesAccordionItem/CaseFilesAccordionItem.tsx

```diff
@@ -81,20 +81,24 @@ const CaseFilesAccordionItem: React.FC<Props> = (props) => {
           </Text>
           {canCaseFilesBeUploaded() && (
             <AnimatePresence>
-              {uploadState === UploadState.UPLOAD_ERROR && (
+              {(uploadState === UploadState.NONE_AVAILABLE ||
+                uploadState === UploadState.NONE_CAN_BE_UPLOADED ||
+                uploadState === UploadState.UPLOAD_ERROR) && (
                 <UploadStateMessage
                   icon="warning"
                   iconColor="red600"
-                  message={formatMessage(m.someFilesUploadedToCourtText)}
-                />
-              )}
-              {uploadState === UploadState.ALL_UPLOADED && (
-                <UploadStateMessage
-                  icon="checkmark"
-                  iconColor="blue400"
-                  message={formatMessage(m.allFilesUploadedToCourtText)}
+                  message={formatMessage(m.someFilesNotUploadedToCourtText)}
                 />
               )}
+              {(workingCase.caseFiles || []).length > 0 &&
+                (uploadState === UploadState.ALL_UPLOADED ||
+                  uploadState === UploadState.ALL_UPLOADED_NONE_AVAILABLE) && (
```

Comment:
Show all uploaded message if all files have been uploaded to Auður. It does not matter if some of the S3 links are broken.